docs/api/api-specification.md

```diff
@@ -0,0 +1,2170 @@
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "7",
+            "text": "headache",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "8",
+            "text": "sore-throat",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "9",
+            "text": "new-loss-of-taste-or-smell",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          }
+        ],
+        "resourceType": "QuestionnaireResponse"
+      }
+    }
+  ],
+  "resourceType": "Bundle"
+}
+```
+</details>
+
+
+<a name="create"/>
+
+### Creating
+
+The API supports creating new monitorees.
+
+#### Extensions
+
+<a name="create-ext"/>
+
+Along with supporting the US Core extensions for race, ethnicity, and birthsex, Sara Alert includes four additional extensions for things specific to the Sara Alert workflows.
+
+Use `http://saraalert.org/StructureDefinition/preferred-contact-method` for specifying the monitorees Sara Alert preferred contact method (options are: `E-mailed Web Link`, `SMS Texted Weblink`, `Telephone call`, `SMS Text-message`, `Opt-out`, and `Unknown`).
+
+```json
+{
+  "url": "http://saraalert.org/StructureDefinition/preferred-contact-method",
+  "valueString": "E-mailed Web Link"
+}
+```
+
+Use `http://saraalert.org/StructureDefinition/preferred-contact-time` for specifying the monitorees Sara Alert preferred contact time (options are: `Morning`, `Afternoon`, and `Evening`).
+
+```json
+{
+  "url": "http://saraalert.org/StructureDefinition/preferred-contact-time",
+  "valueString": "Morning"
+}
+```
+
+Use `http://saraalert.org/StructureDefinition/symptom-onset-date` to specify when the monitoree's first symptoms appeared for use in the Sara Alert isolation workflow.
+
+```json
+{
+  "url": "http://saraalert.org/StructureDefinition/symptom-onset-date",
+  "valueDate": "2020-05-23"
+}
+```
+
+Use `http://saraalert.org/StructureDefinition/last-exposure-date` to specify when the monitoree's last exposure occurred for use in the Sara Alert exposure workflow.
+
+
+```json
+{
+  "url": "http://saraalert.org/StructureDefinition/last-exposure-date",
+  "valueDate": "2020-05-18"
+}
+```
+
+Use `http://saraalert.org/StructureDefinition/isolation` to specify if the monitoree should be in the isolation workflow (omitting this extension defaults this value to false, leaving the monitoree in the exposure workflow).
+
+```json
+{
+  "url": "http://saraalert.org/StructureDefinition/isolation",
+  "valueBoolean": false
+}
+```
+
+#### POST `[base]/Patient`
+
+<a name="create-post-pat"/>
+
+To create a new monitoree, simply POST a FHIR Patient resource.
+
+##### Request Body
```

Comment:
It looks like we have some inconsistent styling of these headers. For example, here, `Request body` has `####`, but line 1126, we only use 3, `###`. This is more noticeable now, since the different headers are styled differently: 
![image](https://user-images.githubusercontent.com/52747882/102382047-f8b2a180-3f97-11eb-9b34-c7538a93cef5.png)

Reply:
It seemed we were using `#` and `###` on this page, but not `##`, and so I just "promoted" every header by one level, and I think it looks better visually now. Also, in the couple cases where "Request Body" or "Response" were formatted as `####` headers, I just changed those to regular bolded text, because I didn't want them in the TOC. Let me know if you think it looks ok now. https://github.com/SaraAlert/SaraAlert/pull/598/commits/490bef24961bf80cb728c31cd3185f0d063a4d0b

Reply:
Yeah I think it parses visually really well now.